frontend/dogplus-frontend/src/components/CreatePostButton.tsx

```diff
@@ -102,7 +112,28 @@ export const CreatePostButton: React.FC<CreatePostButtonProps> = ({
                     </Button>
                   </div>
                 )}
-                {!postImage && (
+                {postVideo && (
```

Comment:
Return toast error when file is not in correct format and does not match a image or video